### PR TITLE
fix(test):update tests to fix iteration kebab

### PR DESCRIPTION
**Pull request by rgarg1**
Detection of kebab menu for iterations was failing due to some recent changes; fixed it.

Also made a change to protractor_config to make tests run in a particular workspace and not in the primary space, which caused interruption.

### CODE REVIEW

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 1]
```diff
@@ -608,7 +608,7 @@ class WorkItemListPage {
 
   /* Iterations Page object model */
   clickIterationKebab (index){
-    return element(by.xpath ("(.//button[@type='button'])["+ index +"]")).click();
+    return element(by.xpath ("(.//button[@class='btn btn-link dropdown-toggle']/i[@class='fa fa-ellipsis-v'])["+ index +"]")).click();
```
Comment (ldimaggi) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: This is fine for the short-term, but can you see about adding a unique ID for the button? Thx!

Comment (rgarg1) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: Agree.

**Comment by fabric8cd** [comment]
@rgarg1 fabric8/fabric8-ui:SNAPSHOT-PR-2083-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2083-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
Thanks @nimishamukherjee

**Pull request merged**